lib/fortnox/api/environment_validation.rb

```diff
@@ -12,21 +12,27 @@ class CircularQueue
 
         def initialize *items
           @queue = [ *items ]
```

Comment:
No that would be weird. It's set in the constructor, instance level operation, but if it is class level the contents of that list could change when another instance is created and your counter could end up outside the new lists end ... We could argue that all of the state in that class should be class level, but how to you set the list then?